tokenbucket.go

```diff
@@ -332,7 +332,7 @@ const (
 )
 
 func redisKey(prefix, key string) string {
-	return fmt.Sprintf("%s/%s", prefix, key)
+	return fmt.Sprintf("{%s}%s", prefix, key)
```

Comment:
Please add a comment explaining why the hash tag is necessary here.

Reply:
Sure. Copied the PR description into the code as the comment, with hyperlink pointing to Redis docs online.